docs/source/contributing/tests.rst

```diff
@@ -3,18 +3,17 @@
 ===================================
 Testing JupyterHub and linting code
 ===================================
-
-Unit tests help confirm that JupyterHub works as intended, including after modifications are made. Additionally, they help in clarifying our expectations for our code.
+Unit test helps to confirm that JupyterHub works as it is supposed to work even when the modifications are made . Additionally they clarify us about our expectations from the code. 
```

Comment:
I don't thihnk the changes here are correct - It should stay `unit tests`. "Works as intended" is a standard phrase we should keep. There also shouldn't be spaces added before periods.